packages/web_benchmarks/lib/src/recorder.dart

```diff
@@ -238,7 +238,7 @@ abstract class SceneBuilderRecorder extends Recorder {
     final Completer<Profile> profileCompleter = Completer<Profile>();
     _profile = Profile(name: name);
 
-    window.onBeginFrame = (_) {
+    PlatformDispatcher.instance.onBeginFrame = (_) {
```

Comment:
This API was introduced almost three years ago (https://github.com/flutter/engine/pull/20496) and has been on the stable channel for a long time. Migrating to it directly will not scarify backwards compatibility to any of the versions we care about.